### ISSUE TITLE
Feature request: support config values

### ISSUE THREAD

**Issue by marcinant** [jtatarik/magit-gitflow#12]
magit-gitflow does not respect preconfigured values of flags.

For example: I added  gitflow.release.finish.push = yes to git configuration.
Magit-gitflow does not test this value and --push option is inactive, while it should be marked as active automatically.

**Comment by jtatarik** [comment]
You're right. I was not aware of the flags' existence, so they are completely ignored by magit-gitflow. Will fix.
